e2e/fake_remote_repo/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@aspect_rules_sol//sol:defs.bzl", "sol_sources")
+
+REMAP_TARGET = "./external/fake_remote_repo/"
+
+sol_sources(
```

Comment:
need to add this e2e to https://github.com/aspect-build/rules_sol/blob/main/.github/workflows/ci.yaml#L24

Reply:
Turned into a bit of a yak-shaving exercise with `.bazelrc` missing then the need for a no-op test :smile: